language/compiler/ir_to_bytecode/src/compiler.rs

```diff
@@ -26,13 +27,56 @@ use std::{
 use vm::{
     access::ModuleAccess,
     file_format::{
-        self, Bytecode, CodeUnit, CompiledModule, CompiledModuleMut, CompiledProgram,
+        self, Bytecode, CodeOffset, CodeUnit, CompiledModule, CompiledModuleMut, CompiledProgram,
         CompiledScript, CompiledScriptMut, FieldDefinition, FieldDefinitionIndex,
         FunctionDefinition, FunctionSignature, Kind, LocalsSignature, MemberCount, SignatureToken,
         StructDefinition, StructFieldInformation, StructHandleIndex, TableIndex,
     },
 };
 
+macro_rules! record_srcloc_info {
```

Comment:
This feels a bit wordy to me. Thoughts on `record_loc` ? or `record_src_loc`? 

Just `srcloc_info` feels a bit redundant?